css/sass/_header.scss

```diff
@@ -0,0 +1,68 @@
+.blog-header {
+  h1 a {
+    color: #fff;
+  }
+  form {
+    color: #444;
+  }
+  select {
+    width: 100%;
+    margin-bottom: 0.3em;
+  }
+  .follow-us a img {
+    margin: 0;
+    width: 40px;
+  }
+  article {
+    height: 200px;
+    background-position: center center;
+    background-size: cover;
+    .entry-header {
+      background: none;
+    }
+    .entry-content {
+      overflow: hidden;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      background: #000;
+      background: rgba(0,0,0,.66);
+      color: #ddd;
+      width: 100%;
+      padding: 20px;
+    }
+  }
+  .search-wrapper {
+    position: relative;
+    display: block;
+    label {
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      margin: -1px;
+      padding: 0;
+      overflow: hidden;
+      clip: rect(0,0,0,0);
+      border: 0;
+    }
+    input[type=submit] {
```

Comment:
Just for consistency it's probably best to use either the quoted or non-quoted type attributes (both are currently used in the SASS)

Reply:
quotes added